packages/web-components/src/radio-group/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './radio-group.js';
+export { definition as RadioGroupDefinition } from './radio-group.definition.js';
+export { styles as RadioGroupStyles } from './radio-group.styles.js';
+export { template as RadioGroupTemplate } from './radio-group.template.js';
```

Comment:
I'm not sure if it needs to be done in this file, but given `orientation` is an attribute here, I think we will want to export the specific namespaced type for the Radio Group orientation.